tests/apollo/util/bft.py

```diff
@@ -396,6 +396,15 @@ def all_replicas(self, without=None):
 
         return list(set(range(0, self.config.n)) - without)
 
+    def random_set_of_replicas(self, size, without=None):
```

Comment:
Hi Yoni! Hristo told me you were going to extract this method as a utility in BftTestNetwork. Could you also refactor this occurrence too:
https://github.com/vmware/concord-bft/blob/master/tests/apollo/test_skvbc_network_partitioning.py#L264

Thanks!

Reply:
done

Reply:
Thanks Yoni!